.travis.yml

```diff
@@ -2,12 +2,36 @@ language: ruby
 
 rvm:
 - 1.9.3
-- jruby-19mode
+- jruby-9.1.9.0-d19
```

Comment:
Yeah, it's 19mode but on a newer engine. (To avoid that issue I mentioned above, which affected both jruby modes)